packages/envd/spec/filesystem/filesystem.proto

```diff
@@ -91,6 +92,7 @@ message WatchDirResponse {
 
 message CreateWatcherRequest {
     string path = 1;
+    bool recursive = 2;
```

Comment:
same comment as for the SDK PR, do you think it would make sense to not add this option and always watch recursively, and also as to avoid changing the communication body too much.

Reply:
Yes, I think so. But as per the instructions

> make it so that the updated version is backward compatible

changing the behavior might introduce a breaking change for current users. I was thinking, that this parameter could be eventually changed to true by default and then deprecated and removed.

But if you think that this kind of change is on par with other changes you do in between versions, then I’m fine with removing non-recursive watching.

Reply:
I read that as backward compatible as far as the API goes, but you raise a good point about what our users expect in terms of behaviour. Do you think that given that these SDKs are versioned and that from this version on we explicitly state in the docs and comments that this is recursive could help?

Reply:
Yeah, I prefer having the parameter added in the API, adding the recursive flag to SDK methods, and then when we do breaking SDK update, changing the default behavior to `recursive=True` while you can still specify it.